edk2 build fails

edk2 build fails.

Here is the error log
```
------
 > [5/7] RUN git clone --recursive https://github.com/tianocore/edk2.git edk2  && (cd edk2 && git checkout 38c8be123aced4cc8ad5c7e0da9121a181b94251)  && make -C edk2/BaseTools/Source/C:
#8 114.2 /usr/bin/ld: /home/vscode/edk2/BaseTools/Source/C/BrotliCompress/brotli/c/enc/fast_log.h:53: undefined reference to `kBrotliLog2Table'
#8 114.2 /usr/bin/ld: /home/vscode/edk2/BaseTools/Source/C/BrotliCompress/brotli/c/enc/fast_log.h:53: undefined reference to `kBrotliLog2Table'
#8 114.2 /usr/bin/ld: brotli/c/enc/metablock.o:/home/vscode/edk2/BaseTools/Source/C/BrotliCompress/brotli/c/enc/fast_log.h:53: more undefined references to `kBrotliLog2Table' follow
#8 114.2 /usr/bin/ld: ../bin/BrotliCompress: hidden symbol `kBrotliCopyExtra' isn't defined
#8 114.2 /usr/bin/ld: final link failed: bad value
#8 114.2 collect2: error: ld returned 1 exit status
#8 114.2 make[1]: *** [../Makefiles/app.makefile:18: ../bin/BrotliCompress] Error 1
#8 114.2 make[1]: Leaving directory '/home/vscode/edk2/BaseTools/Source/C/BrotliCompress'
#8 114.2 make: Leaving directory '/home/vscode/edk2/BaseTools/Source/C'
#8 114.2 make: *** [GNUmakefile:76: BrotliCompress] Error 2
------
error: failed to solve: executor failed running [/bin/sh -c git clone --recursive https://github.com/tianocore/edk2.git edk2  && (cd edk2 && git checkout 38c8be123aced4cc8ad5c7e0da9121a181b94251)  && make -C edk2/BaseTools/Source/C]: exit code: 2
Error: Process completed with exit code 1.
```

--- a/amd64/Dockerfile
+++ b/amd64/Dockerfile
@@ -88,7 +88,7 @@ WORKDIR /home/${USERNAME}
 
 # build EDK II
 RUN git clone --recursive https://github.com/tianocore/edk2.git edk2 \
- && (cd edk2 && git checkout 38c8be123aced4cc8ad5c7e0da9121a181b94251) \
+ && (cd edk2 && git checkout --recurse-submodules 38c8be123aced4cc8ad5c7e0da9121a181b94251) \
  && make -C edk2/BaseTools/Source/C
 #  && rm -rf edk2
 

--- a/arm64/Dockerfile
+++ b/arm64/Dockerfile
@@ -92,7 +92,7 @@ WORKDIR /home/${USERNAME}
 
 # build EDK II
 RUN git clone --recursive https://github.com/tianocore/edk2.git edk2 \
- && (cd edk2 && git checkout 38c8be123aced4cc8ad5c7e0da9121a181b94251) \
+ && (cd edk2 && git checkout --recurse-submodules 38c8be123aced4cc8ad5c7e0da9121a181b94251) \
  && make -C edk2/BaseTools/Source/C
 
 # clone mikanos devenv

